packages/shared/src/graphql/search.ts

```diff
@@ -289,3 +289,13 @@ export const sendSearchQuery = async (
 
   return new EventSource(`${searchQueryUrl}?${params}`);
 };
+
+export type SearchSuggestion = {
```

Comment:
Just nitpicking why this is under the graphql layer, but probably just what we had already?

Reply:
Its result from search suggestions query, from what I have seen there was no static type for it, only runtime in different places.